static/img/spectrum-workflow-custom.svg

```diff
@@ -0,0 +1,24 @@
+<svg id="icon" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We shouldn't be using `id` attributes on these. `id` should be unique on the page. If needed for styling purposes you can address this via class name or descendant selector when moving into the React component.

Reply:
The id attribute was needed for the `<use>` tag. Unnecessary now that the SVG is inlined